layers/+lang/html/packages.el

```diff
@@ -155,7 +155,7 @@
      'smartparens-mode)
    '(css-mode-hook scss-mode-hook sass-mode-hook less-css-mode-hook))
 
-  (add-hook 'web-mode-hook 'spacemacs/toggle-smartparens-off))
+  (add-hook 'web-mode-hook 'turn-off-smartparens-mode))
```

Comment:
Should it be? If so, I can move it there. How should layers handle cross-package dependencies like this? I've asked myself this in my personal configurations, where I would like to enable or disable some minor mode in a particular major mode. Generally, should the `add-hook` be in the major mode `post-init-<major-mode>` function, or in the minor mode's `post-init-<minor-mode>`?